libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -356,6 +356,15 @@ void AP_Vehicle::setup()
 #endif
 
 #if AP_SERIALMANAGER_ENABLED
+#if HAL_WITH_IO_MCU && (HAL_UART_IOMCU_IDX < HAL_UART_NUM_SERIAL_PORTS)
+    if (BoardConfig.io_enabled()) {
+        // If IO is enabled, we need to set the protocol to None
+        // as the IO MCU UART is handled by the AP_IOMCU class
+        serial_manager.set_protocol_and_baud(HAL_UART_IOMCU_IDX, AP_SerialManager::SerialProtocol_None, 0);
```

Comment:
SerialProtocol_IOMCU would make more sense

Reply:
changed to SerialProtocol_IOMCU